src/Searchlight/DataSource.cs

```diff
@@ -128,6 +130,19 @@ private void AddInclude(string name, object incl)
             _includeDict[upperName] = incl;
         }
 
+        private static bool CheckPresence(string sort, PropertyInfo[] props)
+        {
+            foreach (var prop in props)
+            {
+                if (String.Equals(sort, prop.Name, StringComparison.InvariantCultureIgnoreCase))
+                {
+                    return true;
+                }
+            }
+
+            return false;
```

Comment:
You can replace this entire block of code with a LINQ expression!

`return (from prop in props where String.equals(sort, prop.name, StringComparison.InvariantCultureIgnoreCase) select prop).Any();`